Task3/src/main/java/Building.java

```diff
@@ -0,0 +1,108 @@
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Please add pom.xml to this project. Inside the pom.xml, set your java version. This will allow me to load your project into my IDE without any additional steps.
And add a .gitignore file that points to files that are specific to your IDE and not specific to your project. For example, the .idea folder or files with the iml extension.